lib/multiply.rb

```diff
@@ -1,2 +1,15 @@
 def multiply(num1, num2)
+    
+    i = 0
+    answer = 0
+    
+    while(i<num1)
+    
+    answer = answer + num2
```

Comment:
For both lines 8 and 9, what is an operator that we can use as shorthand when we are executing some code like `a_var = a_var + another_var`?